src/behaviors/ElementStatusBehavior.php

```diff
@@ -23,18 +19,14 @@ class ElementStatusBehavior extends Behavior
      */
     public $statusBeforeSave = '';
 
-    /**
-     * @var bool
-     */
-    public $statusChanged = false;
 
     // Public Methods
     // =========================================================================
 
     /**
      * Saves the status of an element before it is saved
      */
-    public function onBeforeSaveStatus()
+    public function rememberPreviousStatus()
```

Comment:
> Why renaming the method names?
> `onBeforeSaveStatus` to `rememberPreviousStatus`
> `onAfterSaveStatus` to `fireEventOnChange`

From outside it's clear what's happening inside. Compare:

`$element->onBeforeSaveStatus()` vs `$element->rememberPreviousStatus()`
`$element->onAfterSaveStatus()` vs `$element->fireEventOnChange()`